### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Helenerineium**
to reflect build script changes
Also update vapoursynth to R47.2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ For information about the compiler environment see the wiki, there you also have
     - mplayer (svn) (unsupported)
     - mpv (git) including in addition to ffmpeg libs:
         - Base build (ffmpegChoice=2 or 3)
-            - ANGLE (from <https://i.fsbn.eu/pub/angle/>)
+            - ANGLE (git)
```
Comment (wiiaboo) [inline README.md]:
```suggestion
            - ANGLE Headers (git)
                - requires ANGLE shared libraries from somewhere else (i.e. Chrome, Firefox) for gpu-context=angle support
```

**Comment by wiiaboo** [comment]
The commits changing the vapoursynth version should be separate from the others for easier reverting.
And they also should have proper names, not Github Web UI default names.

File: README.md [hunk 2]
```diff
@@ -50,7 +51,6 @@ For information about the compiler environment see the wiki, there you also have
             - gnutls (latest release)
         - libaom (git)
         - libass (git)
-            - by default with DirectWrite backend
```
[the remark this answers is not in the document]
Comment (Helenerineium) [reply README.md]: Mixed with harfbuzz

**Comment by wiiaboo** [comment]
Merged in e10629276abe904a1a81d7901a858f66c55b4eb5

**Pull request closed without merging**